nservicebus/upgrades/supported-platforms.md

```diff
@@ -15,6 +15,8 @@ related:
 | .NET Core | 3.1 (LTS) | [Windows / Linux](https://github.com/dotnet/core/blob/master/release-notes/3.1/3.1-supported-os.md) | Supported | macOS is supported only for development purposes. |
 | .NET | 5.0 | [Windows / Linux](https://github.com/dotnet/core/blob/main/release-notes/5.0/5.0-supported-os.md) | Supported | macOS is supported only for development purposes. |
```

Comment:
```suggestion
| .NET | 5.0 | [Windows / Linux](https://github.com/dotnet/core/blob/main/release-notes/5.0/5.0-supported-os.md) | Supported | macOS is supported only for development purposes. |
| .NET | 6.0 Preview | [Windows / Linux](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-6/) | [Current Status](Link to discourse announcement goes here)  |  |
```

Reply:
This document https://github.com/dotnet/core/blob/main/release-notes/6.0/supported-os.md already exists for net6, that's probably what we should link the Windows / Linux column to